app/lib/activitypub/activity/create.rb

```diff
@@ -67,7 +67,7 @@ def process_status_params
         visibility: visibility_from_audience,
         thread: replied_to_status,
         conversation: conversation_from_uri(@object['conversation']),
-        media_attachment_ids: process_attachments.take(4).map(&:id),
+        media_attachment_ids: process_attachments.map(&:id),
```

Comment:
Could you turn this into a PR that does not process more than 4 attachments? Most clients won't be able to display more than 4 right now, and the important thing is not storing stuff we don't use.